### ISSUE TITLE
Question about branches and PR

### ISSUE THREAD

**Issue by liusida** [daniel-lewis-ab/litegraph.js#21]
Hi guys, 

I saw that @daniel-lewis-ab is working on the `reversion` and @atlasan is working on `forwardcompatible`. And I've sumitted a PR to the `master`. Is `master` the branch I should PR to? Did I do it right? Or, should I join @atlasan to fix things on `forwardcompatible`? 

I need some suggestion. Thanks!

**Comment by liusida** [comment]
> Hi @liusida, good to see you here ^ _ ^ I chose the name for the temporary branch I'm working on. Surely the intention is to be compatible and forward looking ^ _ ^ Please feel free to join, need some hands in testing, cleaning, and improving.

Sounds like a plan. Do you have any specific tasks that I can work on? I can try one of them first.

**Comment by liusida** [comment]
And, what do you think about the PR #19, since that PR can make litegraph.js runnable in plain html, and we won't need to start a http server for the development any more. Should I PR to your branch instead of the `master`?

**Comment by liusida** [comment]
> making workflows that are tests themselves

What a brilliant suggestion~ xD

**Comment by daniel-lewis-ab** [comment]
Any features you want to add are good on forwardcompatible, that's
basically forked off of reversion AFAIK.

My thought is that we're going to get reversion caught up, and then start
porting over any features we want one at a time so that there's some
discipline in how they're integrated while still allowing us to dev
freely.  Both Atlasan and I have trouble doing just one thing at a time so
the commits and versioning are clean.

**Comment by liusida** [comment]
> think about the best way to allow different version of the library in ComfyUI ( I was imagining a setting in there where to specify the name of the folder/file to be included. This would help testing and switching to pre-release or custom UIs easily. eg. clone the repo in a folder and specify that folder name in the ComfyUI settings

Yeah. I naively thought that compiling this project into a `litegraph.core.js` file and replace [the original one in ComfyUI](https://github.com/comfyanonymous/ComfyUI/blob/master/web/lib/litegraph.core.js) would automatically bring new features to ComfyUI. (I've actually tried, but it doesn't work...)

How do you do this manually @atlasan ? Can the current `forwardcompatible` branch be applied to current ComfyUI?
